docsbox/docs/tasks.py

```diff
@@ -78,12 +79,15 @@ def process_document_convertion(path, options, meta, current_task):
             force_ocr += 1
     else:
         with TemporaryDirectory() as tmp_dir:
-            os.mkdir(os.path.join(tmp_dir, "user"))
-            shutil.copyfile("/home/config/registrymodifications-SelectPdfVersion-" + options.get("output_pdf_version", None) + ".xcu", os.path.join(tmp_dir, "user/registrymodifications.xcu"))
-
             if options["format"] in app.config[app.config["CONVERTABLE_MIMETYPES"][meta["mimetype"]]["formats"]]:
-                run(['soffice', '-env:UserInstallation=file://' + tmp_dir, '--headless', '--infilter=' + app.config["CONVERTABLE_MIMETYPES"][meta["mimetype"]]["name"], '--convert-to', options["format"], '--outdir', app.config["MEDIA_PATH"], path])
+                filter = ''
+                if (options["format"] == "pdf"):
+                    app.config["PDF_EXPORT_PARAMS"]["SelectPdfVersion"]["value"] = options["output_pdf_version"]
```

Comment:
Do we need to update app.config? Rather use a temp variable for the PDF_EXPORT_PARAMS dict and set the pdf version there before dumping the json